ipatests/test_integration/test_replica_promotion.py

```diff
@@ -484,6 +484,18 @@ def test_replica_not_marked_as_renewal_master(self):
             "Replica hostname found among CA renewal masters"
         )
 
+    def test_renewal_replica_with_ipa_ca_cert_manage(self):
+        """Make replica as IPA CA renewal master using
+        ipa-cacert-manage --renew"""
+        master = self.master
+        replica = self.replicas[0]
+        self.assertCARenewalMaster(master, master.hostname)
+        replica.run_command([paths.IPA_CACERT_MANAGE, 'renew'])
+        self.assertCARenewalMaster(replica, replica.hostname)
+        # set master back to ca-renewal-master
+        master.run_command([paths.IPA_CACERT_MANAGE, 'renew'])
+        self.assertCARenewalMaster(master, master.hostname)
```

Comment:
I would also check how other server (replica in this case) sees the situation. So:

`self.assertCARenewalMaster(master, master.hostname)`
`self.assertCARenewalMaster(replica, master.hostname)`